helm/alfresco-content-services/charts/alfresco-search/requirements.yaml

```diff
@@ -0,0 +1,6 @@
+# Alfresco Insight Engine brings Alfresco Insight Zeppelin
```

Comment:
now the requirements are embedded this file can be deleted

Reply:
deleted requirements file